be/src/olap/rowset/segment_v2/page_decoder.h

```diff
@@ -81,13 +81,20 @@ class PageDecoder {
     // allocated in the column_vector_view's mem_pool.
     virtual Status next_batch(size_t* n, ColumnBlockView* dst) = 0;
 
+    // Same as `next_batch` except for not adding index
+    virtual Status peek_next_batch(size_t *n, ColumnBlockView* dst) {
```

Comment:
Why need this interface？

Reply:
In ListColumnReader, we want read one next row and do not want to change the cursor of page decoder.

Reply:
OK. Please add more comment.